packages/host/app/commands/create-card-instance.ts

```diff
@@ -0,0 +1,42 @@
+import { service } from '@ember/service';
+
+import { type LooseSingleCardDocument } from '@cardstack/runtime-common';
+
+import type * as BaseCommandModule from 'https://cardstack.com/base/command';
+
+import HostBaseCommand from '../lib/host-base-command';
+
+import CardService from '../services/card-service';
+
+export default class CreateCardInstanceCommand extends HostBaseCommand<
+  typeof BaseCommandModule.CreateInstanceInput
+> {
+  @service declare private cardService: CardService;
+
+  description = `Create a new card instance given a card ref and realm.`;
+
+  async getInputType() {
+    let commandModule = await this.loadCommandModule();
+    const { CreateInstanceInput } = commandModule;
+    return CreateInstanceInput;
+  }
+
+  protected async run(
+    input: BaseCommandModule.CreateInstanceInput,
+  ): Promise<undefined> {
+    if (!input.module || !input.realm) {
+      throw new Error(
+        "Create instance command can't run because it doesn't have all the fields in arguments returned by open ai",
+      );
+    }
+    let doc: LooseSingleCardDocument = {
+      data: {
+        meta: {
+          adoptsFrom: input.module,
+          realmURL: input.realm,
+        },
+      },
+    };
+    await this.cardService.createFromSerialized(doc.data, doc);
```

Comment:
we should really be using the CardResource to make cards, not the CardService directly (it would be great if we could deprecate this so others don't fall into this trap). When you make a card with the CardService directly that means that the card will not be part of the identity map and will not receive live updates, as well as other realy bad side effects.